### DISCUSSION TITLE
总是 Heartbeat failed 重连

### DISCUSSION THREAD

**Discussion by FoliageOwO** [Mrs4s/go-cqhttp#756] [Q&A] [answered]
go-cqhttp 挂上一段时间就提示 `Bot已离线 (Heartbeat failed: Packet timed out)，尝试重连`，每隔大约半个小时就会这样，前后半分钟无法接收到消息，很影响使用。
我看了一下相关的 issues，但是没有搞懂最后是什么意思。是 go-cqhttp 的原因还是我的网络原因？最后又该怎么解决呢...

**Comment by Mrs4s** [comment, marked as answer]
这个应该是网络问题
